dcor/_energy.py

```diff
@@ -68,6 +79,8 @@ def energy_distance(x, y, **kwargs):
         variables while the rows are individual instances of the random vector.
     exponent: float
         Exponent of the Euclidean distance, in the range :math:`(0, 2)`.
+    average: Callable[[ArrayLike], float]
```

Comment:
In this function the `average` parameter is missing from the first line of the docstring too.